docs/src/enterprise/configuration.md

```diff
@@ -116,6 +116,26 @@ Configuration section for authenticating to lakeFS using AWS presign get-caller-
 * `auth.external.aws_auth.http_client.timeout` `(duration : 10s)` - The timeout for the HTTP client used to communicate with AWS STS
 * `auth.external.aws_auth.http_client.skip_verify` `(bool : false)` - Skip SSL verification with AWS STS
 
+## auth.prefetched_cache
+
+Configuration section for the prefetched cache
+The prefetched cache is used to cache entries from the auth KV store in memory to reduce the number of calls to the KV store.
+In addition to the prefetched cache, there is also an LRU negative cache that caches negative lookups (i.e. entries that do not exist in the KV store).
```

Comment:
less implementation details and more functional.
the reduce in kv calls during authorization requests, but will perform more calls in the background as it will read the auth every minute.